### PR TITLE
Changelog Update 07/05/2023

### PR DESCRIPTION
**Pull request by Ollieollieolllie**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -48,19 +148,10 @@
 * Updated Hellfire Penetrator (#135)
 * Move engine simulation to be local but globally synchronised. (#134)
 
-*15/11/2021*
-* Moved sight select to shift
-
-*07/11/2021*
```
Comment (mattysmith22) [inline Changelog.md:54]: Why were these removed?

**Review by mattysmith22 — APPROVED** [review]
Great start on this, I particularly like the PR numbers next to the changes.

One important thing to think about is the audience of this document, it isn't for devs but for the users of our mod. This means that they might not necessarily know what "Mpd-Ase-Page-Update" means or care that our pbos use tabs rather than spaces.

Each line should start with a verb, so "Added", "Replaced", "Removed", and "Updated" are all great starts. They should be clear and should be understandable by someone with reasonable experience in our mod

**Review by mattysmith22 — CHANGES_REQUESTED** [review]
Meant to mark as request changes, apologies for confusion

**Review by mattysmith22 — CHANGES_REQUESTED** [review]
Meant to mark as request changes, apologies for confusion

File: Changelog.md (hunk 1 above)
Comment (Ollieollieolllie) [reply Changelog.md:54]: Because they were not ors but commits from prs

**Comment by Ollieollieolllie** [comment]
So remove and edit updates to be important definable objectives met?

**Review by mattysmith22 — APPROVED** [review]
